src/ReactiveUI/ReactiveUI.csproj

```diff
@@ -10,23 +10,23 @@
   <ItemGroup>
     <Compile Remove="Platforms\**\*.cs" />
     <None Include="Platforms\**\*.cs" />
-    <PackageReference Include="System.Reactive" Version="3.1.0" />
+    <PackageReference Include="System.Reactive" Version="4.0.0-preview00001" />
     <PackageReference Include="Splat" Version="2.0.0" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.1' ">
-    <Compile Include="Platforms\netstandard1.1\**\*.cs" />
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.3' ">
+    <Compile Include="Platforms\netstandard1.3\**\*.cs" />
     <PackageReference Include="System.ComponentModel" Version="4.3.0" />
     <PackageReference Include="System.Diagnostics.Contracts" Version="4.3.0" />
     <PackageReference Include="System.Dynamic.Runtime" Version="4.3.0" />
     <PackageReference Include="System.Runtime.Serialization.Primitives" Version="4.3.0" />
   </ItemGroup>
   
-  <ItemGroup Condition="'$(TargetFramework)' == 'net45'">
-    <Compile Include="Platforms\net45\**\*.cs" />
+  <ItemGroup Condition="'$(TargetFramework)' == 'net452'">
+    <Compile Include="Platforms\net452\**\*.cs" />
```

Comment:
nah there's some wierdness with the system.reactive -preview000 package on NuGet. Spoken with Oren and I'm bumping us up to the nightly from MyGet. If this is 👍 from us then he will promote the System.Reactive build to NuGet.